test/common/firefox-cdp-driver.js

```diff
@@ -91,22 +91,25 @@ function setupLogging(client) {
 
     function processException(info) {
         let details = info.exceptionDetails;
+        if (details.exception)
+            details = details.exception;
+
         // don't log test timeouts, they already get handled
-        if (details.exception && details.exception.className === "PhWaitCondTimeout")
+        if (details.className === "PhWaitCondTimeout")
```

Comment:
Note that our current tasks container use nightly firefox (86.0a1 right now). I'd really like this to work with the current release (84) as well -- did you try this locally? I assume you did, but I'd like to clarify if at least for some time we'll need both code paths.

Reply:
Oh, silly me -- it's right there in line 94.

Reply:
Good point, I forgot that we *have* to use nightly due to releases not having the devtools driver.